.github/workflows/e2e-multicluster-ci.yml

```diff
@@ -120,12 +120,11 @@ jobs:
           ca=$(kubectl get -n default secret "$name" -o go-template='{{index .data "ca.crt" | base64decode}}')
 
           kubectl config use-context k3d-downstream
-          helm -n fleet-system upgrade --install --create-namespace --wait fleet-agent charts/fleet-agent \
+          helm -n cattle-fleet-system upgrade --install --create-namespace --wait fleet-agent charts/fleet-agent \
             --set-string labels.env=dev \
             --set apiServerCA="$ca" \
             --set apiServerURL="https://172.18.0.1.omg.howdoi.website:6443" \
             --set clusterNamespace="fleet-local" \
-            --set systemRegistrationNamespace="fleet-clusters-system" \
```

Comment:
This is being deleted, because the default is fine?

Reply:
I have changed the default so no additional user interaction for new clusters is necessary.